modules/post/multi/gather/firefox_creds.rb

```diff
@@ -142,26 +142,26 @@ def do_decrypt
         if datastore['RECOVER']
           return unless decrypt_recover_omni(temp_file, org_file)
         else
-          print_warning("If you wish to continue by trying to recover, set the advanced option, RECOVER, to TRUE.")
+          print_warning('If you wish to continue by trying to recover, set the advanced option, RECOVER, to TRUE.')
           return
         end
       elsif !session.fs.file.exist?(@paths['ff'] + org_file)
         print_error("Could not download #{org_file}. File does not exist.")
         return
       end
-    end # session.type == "meterpreter"
+    end
```

Comment:
is this just straight up deleting comments? there's a couple of cases of this happening in this file

Reply:
Good.

This module is a mess of else-after-return nested spaghetti.

Reply:
A lot of linters/formatters will frequently unintentionally delete comments, as the AST generally doesn't contain the comment metadata.

In this case the comment wasn't really needed so it's okay 😄